api/controllers/WebViewController.js

```diff
@@ -50,20 +50,34 @@ module.exports = {
 
   ////////////////////////////////////////////////////////////////////////////
 
+  squad: (req, res) => {
```

Comment:
I wasn't able to reference the `adviceBaseUrl` env variable from the `routes` file where this was originally located (the `locals` object was only accepting literal string values from there), so I moved it here in order to be able to pass in that variable.